<?xml version="1.0"?>
<launch>
  <node pkg="dynamic_obstacle_prediction" type="pointcloud_cluster" name="pointcloud_cluster_node" output="screen">
    <remap from="/pointcloud_in" to="/js/velodyne/lidar_crop" />
    <remap from="/markers_out" to="/cluster_markers" />
    <param name="lidar_frame" value="js/front_lidar_link"/>
    <param name="z_cluster_tolerance" value="6.0"/>
    <param name="xy_cluster_tolerance" value="6.0"/>
    <param name="publish_rate" value="10.0"/>
    <param name="xy_pedding_range" value="10"/>
  </node>
</launch>
